api-docs/api-reference/methods/users/post-forgot-pwd-v2.0.rst

```diff
@@ -0,0 +1,111 @@
+.. _post-forgot-pwd-v2.0:
+
+Forgot password
+~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/RAX-AUTH/forgot-pwd
+
+Use the Forgot password operation to get a password reset token delivered to
+the email address associated with your account.
+
+You can use the forgot password operation to reset the password through the
+Cloud Identity API or Rackspace Cloud user interface.
+
+    - To reset the password by using the API, submit the forgot password
+      request without the portal attribute. After you submit the request, the
+      user receives an email that contains a reset password token with
+      directions to use it in the :ref:`Reset password <post-reset-pwd-v2.0>`
+      operation.
+
+    - To reset the password through the user interface, include the
+      portal attribute in the forgot password API request. After you
+      submit the request, the user receives an e-mail that contains a link to
+      the Rackspace user interface to change the password. The portal attribute
+      is meant for Rackspace use only.
+
+    - The portal attribute is for Rackspace use only.
```

Comment:
Delete last bullet -- you have the sentence in the previous bullet item.  

e-mail ==> email